apps/system/js/statusbar.js

```diff
@@ -451,13 +451,82 @@ var StatusBar = {
   },
 
   _paused: 0,
-  pauseUpdate: function sb_pauseUpdate() {
-    this._paused++;
+
+  _pausedEvents: {
+    utilitytraywillshow: false,
+    utilitytrayopen: false,
+    utilitytraywillhide: false,
+    utilitytrayclose: false
   },
```

Comment:
so the `_pausedEvents`' keys are event types? :)
It's a bit confusing... I'll try to figure out a suggestion.